protocols/kad/src/handler.rs

```diff
@@ -138,9 +147,6 @@ pub struct KademliaHandlerConfig {
 
 /// State of an active outbound substream.
 enum OutboundSubstreamState<TUserData> {
-    /// We haven't started opening the outgoing substream yet.
-    /// Contains the request we want to send, and the user data if we expect an answer.
-    PendingOpen(SubstreamProtocol<KademliaProtocolConfig, (KadRequestMsg, Option<TUserData>)>),
```

Comment:
Nice! I always disliked this one because it is not actually a state of the stream.